install_config/install/docker_registry.adoc

```diff
@@ -544,6 +558,31 @@ In the `<host>/test/busybox` example above, `test` refers to the project name.
 ====
 
 
+== Known Issues
+
+=== Image push errors with scaled registry using shared NFS volume
+
+During a build of an image you may see one of the following errors in the output of `oc build-logs BUILD`:
```

Comment:
Do you want to be specific to builds here? Or should we talk more generically about pushes?

Comma after "During a build of an image"